[2024-11-02T13:33:51.690+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-02T13:33:51.717+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tem.bash_task2 manual__2024-11-02T13:33:46.920704+00:00 [queued]>
[2024-11-02T13:33:51.733+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tem.bash_task2 manual__2024-11-02T13:33:46.920704+00:00 [queued]>
[2024-11-02T13:33:51.733+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2024-11-02T13:33:51.768+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): bash_task2> on 2024-11-02 13:33:46.920704+00:00
[2024-11-02T13:33:51.778+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=520) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-02T13:33:51.780+0000] {standard_task_runner.py:72} INFO - Started process 521 to run task
[2024-11-02T13:33:51.780+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tem', 'bash_task2', 'manual__2024-11-02T13:33:46.920704+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/templatin/template.py', '--cfg-path', '/tmp/tmpo_70hvv1']
[2024-11-02T13:33:51.781+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask bash_task2
[2024-11-02T13:33:51.850+0000] {task_command.py:467} INFO - Running <TaskInstance: tem.bash_task2 manual__2024-11-02T13:33:46.920704+00:00 [running]> on host fe3e688f2b29
[2024-11-02T13:33:51.982+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ade' AIRFLOW_CTX_DAG_ID='tem' AIRFLOW_CTX_TASK_ID='bash_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T13:33:46.920704+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-02T13:33:46.920704+00:00'
[2024-11-02T13:33:51.983+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-02T13:33:52.003+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-02T13:33:52.005+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "12"']
[2024-11-02T13:33:52.015+0000] {subprocess.py:86} INFO - Output:
[2024-11-02T13:33:52.017+0000] {subprocess.py:93} INFO - 12
[2024-11-02T13:33:52.018+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-02T13:33:52.049+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-02T13:33:52.050+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tem, task_id=bash_task2, run_id=manual__2024-11-02T13:33:46.920704+00:00, execution_date=20241102T133346, start_date=20241102T133351, end_date=20241102T133352
[2024-11-02T13:33:52.077+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-02T13:33:52.105+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-02T13:33:52.115+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
